assets/docs/callmemehdi/models/AraBERT/1.md

```diff
@@ -0,0 +1,44 @@
+# Module callmemehdi/arabert/1
+
+AraBERT is an Arabic pretrained lanaguage model based on Google's BERT architechture.
+
+<!-- asset-path: https://gsoctfarabert.web.app/arabert.tar.gz -->
+<!-- task: text-embedding -->
+<!-- fine-tunable: true -->
+<!-- format: saved_model_2 -->
+<!-- license: custom -->
+
+## Overview
+
+These results are obtained from pre-training BERT specifically for the Arabic language in the pursuit of achieving the same success that BERT did for the English language.The performance of AraBERT is compared to multilingual BERT from Google and other state-of-the-art approaches
```

Comment:
Can you also provide information about the model source (linking to https://huggingface.co/aubmindlab/bert-base-arabertv2 would be good). Please confirm the version this is (i.e. v2 base? v2 large? etc)